src/components/home/slider/style.tsx

```diff
@@ -98,6 +108,16 @@ export const ModalImg = styled(Img)`
   width: 30vw;
   height: 95%;
   z-index: 1;
+
+  @media only screen and (min-width: 320px) {
+    width: 70vw;
+  }
+  @media only screen and (min-width: 768px) {
+    width: 30vw;
+  }
+  @media only screen and (min-width: 1200px) {
+    width: 30vw;
+  }
```

Comment:
```suggestion
  @media only screen and (min-width: 768px) {
    width: 30vw;
  }
```

1 is enough for this case.